src/main/java/uk/gov/pay/adminusers/queue/event/EventMessageHandler.java

```diff
@@ -182,6 +182,17 @@ private Map<String, String> getMinimumRequiredPersonalisation(String gatewayAcco
                 "serviceReference", transaction.getReference()));
     }
 
+    private Map<String, String> getMinimumPersonalisationForDisputeLost(DisputeLostDetails details, String parentResourceExternalId) {
```

Comment:
Any reason separate method is needed ? Or maybe rename method to `getPersonalisationForDisputeLost` .
`getMinimumPersonalisationForDisputeLost` indicates the result will be personalised further which is not the case.

Reply:
If we only had one method, then calls for every type of dispute would have to supply a DisputeLostDetails argument.  Happy to rename the second method as suggested though.